### PR TITLE
Adding more results to README

### PR DESCRIPTION
**Pull request by sayakpaul**
Did not find the Release tab of the repository. Hence had to host the weights directly in the repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,8 +21,16 @@ with linear probing on the **CIFAR-10** dataset. Our training logs and encoder w
...
 
 _We note that with further hyperparameter tuning and more epochs of pre-training, we can achieve a better performance
-with linear-probing._  We plan on adding results from a few more experiments that we are conducting. So, keep
-an eye out :)
+with linear-probing._  Below we present some more results:
+
+| Config | Masking<br>proportion | LP<br>performance | Encoder weights<br>& logs |
+|:---:|:---:|:---:|---|
+| Encoder & decoder layers: 3 & 1<br>Batch size: 256 | 0.6 | 44.25% | [Link](https://github.com/ariG23498/mae-scalable-vision-learners/blob/master/assets/44_25.zip) |
+| Do | 0.75 | 46.84% | [Link](https://github.com/ariG23498/mae-scalable-vision-learners/blob/master/assets/46_84.zip) |
```
Comment (ariG23498) [inline README.md:29]: What does Do mean here?

**Review by ariG23498 — COMMENTED** [review]
We currently have `assets` and `encoder_weights_logs` folder. What should be the right thing to do here?

I am thinking merging the PR, downloading the weights from the `assets` folder and releasing the zip files. We would also need to delete the `encoder_weights_logs` folder as well. What do you think?

File: README.md (hunk 1 above)
Comment (sayakpaul) [reply README.md:29]: Same setting.

**Comment by sayakpaul** [comment]
Release the `.tar.gz` stuff under "Releases" and update the links wherever applicable. 

Deleting `encoder_weights_logs folder is fine as long as all the places where it's been mentioned have been taken care of.

**Comment by ariG23498** [comment]
@sayakpaul could you review the changes?

**Pull request merged**